loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
@jhsx 
```
import cycle not allowed in test
package github.com/CloudyKit/jet (test)
       	imports github.com/CloudyKit/jet/loaders/httpfs
       	imports github.com/CloudyKit/jet
```

I'll have to rename the `*_test.go` package and import `jet` from within tests.